.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules
 # Ignore ES5 source directory on github.
 # http://mammal.io/articles/using-es6-today/#getting-started
 lib
+npm-debug.log
+*.DS_Store
+*.sublime-workspace
```

Comment:
I feel your pain. I promise I'll have a look at google and see how everybody can be happy here.